test/conftest.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+from ThreeDiToolbox.datasource.netcdf_groundwater import NetcdfGroundwaterDataSource
+from ThreeDiToolbox.utils.patched_threedigrid import (
```

Comment:
patched: is dat nog nodig met de nieuwste threedigrid release? (Geen idee wat er gepatcht is, trouwens).